apps/commudle-admin/src/app/feature-modules/community-control-panel/components/community-forms-and-surveys/community-forms-list/community-forms-list.component.ts

```diff
@@ -1,20 +1,23 @@
-import { Component, OnInit } from '@angular/core';
+import { Component, OnDestroy, OnInit } from '@angular/core';
 import { ActivatedRoute } from '@angular/router';
 import { faPlusSquare } from '@fortawesome/free-solid-svg-icons';
 import { Cell, Settings } from 'angular2-smart-table';
 import { DataFormsService } from 'apps/commudle-admin/src/app/services/data_forms.service';
 import { IDataForm } from 'apps/shared-models/data_form.model';
+import { ICommunity } from 'apps/shared-models/community.model';
 import { CommunityFormsListActionsComponent } from './community-forms-list-actions/community-forms-list-actions.component';
 import { CommunityFormsListStatsComponent } from './community-forms-list-stats/community-forms-list-stats.component';
+import { Subscription } from 'rxjs';
+import { SeoService } from 'apps/shared-services/seo.service';
```

Comment:
let's use @commmudle/shared-models for ICommunity and SeoService from @commudle/shared-service